pybsn/__init__.py

```diff
@@ -467,6 +468,16 @@ def __repr__(self):
         return "BigDbClient(%s)" % self.url
 
 
+def _normalize(v):
+    """ Helper method to normalize query values """
+    if type(v) == bool:
+        # replace to use JSON type booleans, not Python
+        if v:
+            return "true"
```

Comment:
Given that this uses the new "pseudo" XPath syntax introduced in BSC-19829, it will only work with future BigDB versions. I am wondering if we should normalize `True`/`False` to the the strings `'"true"'` / `'"false"'` (with quotes). Then this would work for existing/shipping BigDB versions as well.